src/components/game.js

```diff
@@ -34,23 +34,158 @@ class Game extends React.Component {
     super(props);
     const { data } = this.props;
     const board = [];
-    for (let i = 0; i < data.site.siteMetadata.size; i += 1) {
-      board[i] = new Array(data.site.siteMetadata.size);
+    for (let i = 0; i < data.site.siteMetadata.boardSize; i += 1) {
```

Comment:
Czy ten fragment kodu inicjalizuje Ci pustą planszę? Może wyciągnąć to do funkcji initBoard, co od razu sugeruje co robi dany fragment kodu.

Reply:
Wsadzone do funkcji initBoard